Framework/DataHandling/inc/MantidDataHandling/LoadNexusMonitors2.h

```diff
@@ -99,6 +99,17 @@ class DLLExport LoadNexusMonitors2 : public API::ParallelAlgorithm {
                         std::map<int, std::string> &monitorNumber2Name,
                         std::vector<bool> &isEventMonitors);
 
+  bool isEventMonitor(::NeXus::File &monitorFileHandle) const;
+  std::size_t sizeOfUnopenedEntry(::NeXus::File &file,
+                                  const std::string &entryName) const;
+  bool eventIdNotEmptyIfExists(
+      ::NeXus::File &monitorFileHandle,
+      std::map<std::string, std::string> const &entries) const;
```

Comment:
If we're going with "east const" then I think we should convert the whole file. Consistency at the file-level is still worth having I think...